matr.php

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Matrix task</title>
+</head>
+
+<body>
+<?php
```

Comment:
You may want to extract the php script in a separate file like here: https://github.com/dftopalova/WEB2020-php/pull/1